src/app/actions/media-wall-custom.ts

```diff
@@ -0,0 +1,59 @@
+import { Action } from '@ngrx/store';
+import { WallHeader, WallBackground, WallGrid, WallCard } from '../models';
+
+/**
+ * For each action type in an action group, make a simple
+ * enum object for all of this group's action types.
+ *
+ * The 'actionTypeCheck' utility function coerces strings into string
+ * literal types and runs a simple check to guarantee all
+ * action types in the application are unique.
+ */
+export const ActionTypes = {
+	WALL_HEADER_CHANGE: '[Media Wall] Media Wall Header Change',
+	WALL_BACKGROUND_CHANGE: '[Media Wall] Media Wall Background Change',
+	WALL_GRID_LAYOUT_CHANGE: '[Media Wall] Media Wall Grid Layout Change',
```

Comment:
What is this property used for? What do you mean by layout change? We have only one layout?